browser-test/src/civiform_admin_question_list.test.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AdminPrograms,
+  AdminQuestions,
+  createTestContext,
+  loginAsAdmin,
+} from './support'
+
+describe('Most recently updated question is at top of list.', () => {
+  const ctx = createTestContext()
+  it('sorts by last updated, preferring draft over active', async () => {
+    const {page, adminPrograms, adminQuestions} = ctx
+
+    await loginAsAdmin(page)
+
+    const questionOne = 'question list test question one'
+    const questionTwo = 'question list test question two'
+    await adminQuestions.addNameQuestion({questionName: questionOne})
+    await adminQuestions.addNameQuestion({questionName: questionTwo})
+
+    // Note: CI tests already have test questions
```

Comment:
Is that true? Each test file starts with empty database. And if you have emptry database you can assume only your questions are present and simplify assertions below.